use-shopping-cart/core/slices/cartSlice.js

```diff
@@ -0,0 +1,144 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { createEntry, updateEntry, removeEntry } from '../Entry'
+
+export const cartInitialState = {
+  lastClicked: '',
+  shouldDisplayCart: false,
+  stripe: null,
+  lastClicked: '',
+  shouldDisplayCart: false,
+  cartDetails: {},
+  totalPrice: 0,
+  cartCount: 0,
+  currency: 'USD',
+  language: 'en-US'
```

Comment:
These should be provided by the user. I would suggest a pattern like: 

```js
function createCartSlice(intialState) {
  initialState = {
    lastClicked: '',
    shouldDisplayCart: false,
    stripe: null,
    lastClicked: '',
    shouldDisplayCart: false,
    cartDetails: {},
    totalPrice: 0,
    cartCount: 0,
    currency: 'USD',
    language: 'en-US',
    ...initialState
  }

  // createSlice omitted
  // return omitted
}
```

Reply:
Ah, I think this answers my other question!